### PR TITLE
SWATCH-489 Add Job and ClowdJobInvocation to run account_config org_id migration

### PR DESCRIPTION
**Pull request by mstead**
https://issues.redhat.com/browse/SWATCH-489
Added a Job and ClowdJobInvokation to migrate org_id into the account_config table. This job will run immediately when deployed.
**Testing**

**FROM THE MAIN BRANCH**

Ensure that the following component is added to your bonfire config:
```
      - name: swatch-tally
        host: local
        repo: /home/mstead/sandbox/swatch/subscriptions/swatch-tally
        path: /deploy/clowdapp.yaml
        parameters:
         REPLICAS: 1
         IMAGE: quay.io/cloudservices/rhsm-subscriptions
         RHSM_RBAC_USE_STUB: "true"
         DEV_MODE: "true"
```

Deploy to EE:
```
$ export NAMESPACE=$(bonfire namespace reserve -d 1h)

$ bonfire deploy -n $NAMESPACE rhsm \
-C rhsm --no-remove-resources=rhsm \
-C swatch-tally --no-remove-resources=swatch-tally \
--optional-deps-method=none \
-i quay.io/cloudservices/rhsm-subscriptions=latest
```

Once completely deployed, insert some opted in accounts:
```
$ oc project $NAMESPACE

$ for a in 6376293 6411176 941133 6349152; do \
oc exec $(oc -n $NAMESPACE get -o name pod | grep rhsm-db | sed -e 's/pod\// /g') -i -t -- psql rhsm-db \
-c "insert into account_config (account_number, sync_enabled, reporting_enabled, opt_in_type, created, updated) values ('$a', true, true, 'JMX', '2022-08-03 15:43:37.737126+00', '2022-08-03 15:43:37.737126+00');"; \
done;
```

Check out this branch and deploy the app again.
```
$ bonfire deploy -n $NAMESPACE rhsm   -C swatch-tally   --no-remove-resources=swatch-tally   --optional-deps-method=none   -i quay.io/cloudservices/rhsm-subscriptions=latest
```

Check the openshift console to ensure that the `swatch-tally-opt-in-org-id-populator` job has run.

Check the DB to ensure that the org_id column has been fully populated.
```
$ oc exec $(oc -n $NAMESPACE get -o name pod | grep rhsm-db | sed -e 's/pod\// /g') -i -t -- psql rhsm-db -c 'select * from account_config;'
```

### CODE REVIEW

File: swatch-tally/deploy/clowdapp.yaml [hunk 1]
```diff
@@ -122,6 +123,29 @@ parameters:
...
+  - name: ORG_ID_POPULATOR_IMAGE_TAG
+    value: latest
+  - name: ORG_ID_POPULATOR_CRON_CPU_REQUEST
+    value: 50m
+  - name: ORG_ID_POPULATOR_CRON_MEMORY_REQUEST
+    value: 512Mi
+  - name: ORG_ID_POPULATOR_CRON_CPU_LIMIT
+    value: 300m
+  - name: ORG_ID_POPULATOR_CRON_MEMORY_LIMIT
+    value: 1Gi
```
Comment (kahowell) [inline swatch-tally/deploy/clowdapp.yaml:143]: This suggests it's a CronJob / has a schedule, please remove `_CRON` from these vars.

File: swatch-tally/deploy/clowdapp.yaml [hunk 2]
```diff
@@ -122,6 +123,29 @@ parameters:
...
+  - name: ORG_ID_POPULATOR_SCHEDULE
+    value: 0 0 * * *
```
Comment (kahowell) [inline swatch-tally/deploy/clowdapp.yaml:145]: Vestigial? Please remove.

**Review by kahowell — CHANGES_REQUESTED** [review]

File: swatch-tally/deploy/clowdapp.yaml (hunk 2 above)
Comment (mstead) [reply swatch-tally/deploy/clowdapp.yaml]: DONE

File: swatch-tally/deploy/clowdapp.yaml (hunk 1 above)
Comment (mstead) [reply swatch-tally/deploy/clowdapp.yaml]: DONE

**Comment by mstead** [comment]
@kahowell FYI something is broken this morning. Trying to determine if it is something related to this PR or a general change to the clowdapp.

**Comment by mstead** [comment]
> @kahowell FYI something is broken this morning. Trying to determine if it is something related to this PR or a general change to the clowdapp.

Issue was due to how I was deploying via bonfire. I've updated the How To Test steps.

**Comment by mstead** [comment]
See JIRA issue for app-interface config changes for the stage environment.

NOTE: Will need to set them appropriately for the production env once swatch-tally is clowderized in that env.

**Review by kahowell — APPROVED** [review]

File: swatch-tally/deploy/clowdapp.yaml [hunk 3]
```diff
@@ -831,6 +855,88 @@ objects:
...
+            - -o
+            - org_id
+            - -t
+            - account_config
+            - -b
+            - $(ORG_ID_POPULATOR_BATCH_SIZE)
+            - --ean-translator-addr
+            - http://${TENANT_TRANSLATOR_HOST}:${TENANT_TRANSLATOR_PORT}
+            - -s
+            - disable
```
Comment (kflahert) [inline swatch-tally/deploy/clowdapp.yaml:884]: @mstead not 100% sure on this but I think we'll have to parameterize the ssl -s argument. The default for stage and prod is 'verify-full' and 'disable' is only EE.

**Pull request merged**